File: src/main/java/zielu/gittoolbox/ui/statusbar/StatusBarActions.java
package zielu.gittoolbox.ui.statusbar;

import com.intellij.openapi.actionSystem.AnAction;
import git4idea.repo.GitRepository;
import zielu.gittoolbox.ui.statusbar.actions.FetchAction;

public interface StatusBarActions {

  static AnAction[] actionsFor(GitRepository repository) {
    return new AnAction[]{
        new FetchAction(repository)
    };
  }
}

File: src/main/kotlin/zielu/gittoolbox/ui/statusbar/StatusBarActions.kt
package zielu.gittoolbox.ui.statusbar

import com.intellij.openapi.actionSystem.AnAction
import git4idea.repo.GitRepository
import zielu.gittoolbox.ui.statusbar.actions.FetchAction

internal object StatusBarActions {
  @JvmStatic
  fun actionsFor(repository: GitRepository): List<AnAction> {
    return listOf(FetchAction(repository))
  }
}
